shark/iree_utils/compile_utils.py

```diff
@@ -90,6 +90,17 @@ def create_dispatch_dirs(bench_dir, device):
                 )
 
 
+def dump_isas(bench_dir):
+  for d_ in os.listdir(bench_dir):
+      if os.path.isdir(f"{bench_dir}/{d_}"):
+          for f_ in os.listdir(f"{bench_dir}/{d_}"):
+              if f_.endswith('.spv'):
+                  os.system(
+                      f"/usr/bin/amdllpc -gfxip 11.0 {bench_dir}/{d_}/{f_} -v > \
```

Comment:
can we just call `amdllpc` so it can be anywhere in your path not just /usr/bin ?

Reply:
amdllpc: command not found, even though /usr/bin is in $PATH. also I have to use sudo /usr/bin/amdllpc ... or else Permission denied

Reply:
Wait I may need to set perms on amdllpc. You shouldn't have to use sudo